guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -36,6 +36,16 @@ Qpid dispatch router
 |8000 |TCP |HTTPS |Operating System installers like Anaconda to download kickstart templates to hosts,
 and for downloading iPXE firmware
 endif::[]
+ifeval::["{Build}" == "foreman-deb"]
+|Port |Protocol |Service |Required for
+|80 |TCP |HTTP |Operating System installers like Anaconda, yum, and, if you use the Katello plug-in, for obtaining Katello certificate
+updates
```

Comment:
Is this some broken line?

Reply:
@lzap this renders fine! believe it or not!